### PR TITLE
SEO-2344_Site_Audit_Issues

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @Aishwarya-Ganesan,

Worked on Site Audit Issues, and merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: angular/GettingStarted/getting-started-with-dotnet-cli.md [hunk 1]
```diff
@@ -542,7 +542,7 @@ dotnet run
 
 * Open the `.csproj` file which is in project folder and then press `Ctrl+F5` to launch the application in a browser.
 
-![](/angular/GettingStarted/Images/spatemplateoutput.png)
+![angular GettingStarted spatemplateoutput](Images/spatemplateoutput.png)
```
Comment (Aishwarya-Ganesan) [inline angular/GettingStarted/getting-started-with-dotnet-cli.md:545]: Is this correct word? - spatemplateoutput

**Review by Aishwarya-Ganesan — COMMENTED** [review]
Commented

File: angular/GettingStarted/getting-started-with-dotnet-cli.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply angular/GettingStarted/getting-started-with-dotnet-cli.md]: Changed

**Comment by gowthamanthavasiyappan** [comment]
rebuild

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**